Tests/package.json

```diff
@@ -2,8 +2,35 @@
   "name": "jsruntimehost",
   "version": "1.0.0",
   "private": true,
+  "scripts": {
+    "build": "webpack",
```

Comment:
Maybe add a `"watch": "webpack -watch",` to make it easier for local dev?

Reply:
Fixed